docs/documentation/gsoc.md

```diff
@@ -5,14 +5,144 @@ slug: /gsoc
 sidebar_label: GSOC
 ---
 
-## 🚧 Under Construction 🚧
-:::caution 🚧 This page is under construction
 
-This section is still being worked on. Check back soon for updates!
+The GSOC (Graffiti Several Owner Chunk) feature allows a **full node** to receive messages from many other nodes using a shared Single Owner Chunk (SOC). This enables real-time messaging over Swarm.
 
-:::
+## Overview
 
+GSOC messages are updates to a pre-mined SOC that lands in the **neighborhood** of the recipient node. Only full nodes receive these updates because light nodes do not sync neighborhood chunks. However, **any node** (light or full) can send GSOC messages.
+
+### GSOC vs PSS
+
+GSOC shares some similarities with PSS - both features allow for full nodes to receive messages from other nodes. 
+
+However, there are several key differences:
+
+- Unlike PSS, **GSOC only needs to mine the target chunk once**—multiple messages reuse it, making it **faster, cheaper**, and **more efficient** for recurring updates.
+- Unlike PSS, **No encryption** is used by default, making it unsuitable for handling sensitive data.
```

Comment:
Maybe add this, but better phrased:

- Subsequent GSOC (SOC) updates are not meant to be retrieved. Double-signing an SOC is undefined behaviour.

Reply:
Hmm, I am not quite sure how this should be phrased? I guess it is something like that the GSOC chunk is never really meant to be retrieved at all? It just serves as a channel for the service node to listen for messages over? In the docs I refer to them as GSOC "updates", but nothing is really being saved and updated is it? GSOC chunks just serve as a channel over which to receive ephemeral messages. Does that make sense?

Reply:
Good